editor.planx.uk/src/@planx/components/shared/Schema/Fields.tsx

```diff
@@ -67,34 +80,30 @@ export const TextFieldInput: React.FC<Props<TextField>> = ({ id, data }) => {
   );
 };
 
-export const NumberFieldInput: React.FC<Props<NumberField>> = ({
-  id,
-  data,
-}) => {
-  const { formik, activeIndex } = useListContext();
+export const NumberFieldInput: React.FC<Props<NumberField>> = (props) => {
```

Comment:
also feel free to ignore: It would probably also be my preference to have all the different inputField components in separate files - I have a hatred of long files (I try to keep to max 150 lines). 🤷

Reply:
This is a really good shout - and I so nearly did it before opening the PR!

Implemented in [d0bdced](https://github.com/theopensystemslab/planx-new/pull/3526/commits/d0bdced2dbf1bc0a269024ae97a767db7527743f)